handlers/BlogArticle/index.js

```diff
@@ -11,10 +11,36 @@ import Hero from 'Hero';
 var {PropTypes} = React;
 
 class BlogArticle extends React.Component {
+
+  constructor(props: mixed, context: mixed): void {
+    super(props, context);
+
+    this.state = {
+      defaultImages: {
+        all: '/public/images/blogimg_all.png',
+        development: '/public/images/blogimg_dev.png',
+        product: '/public/images/blogimg_pro.png',
+        business: '/public/images/blogimg_biz.png',
+        culture: '/public/images/blogimg_cult.png',
+      },
+    };
+
+    this.getDefaultImage = this.getDefaultImage.bind(this);
+  }
+
+  getDefaultImage(tags: Array): String {
+    var defaultImages = this.state.defaultImages;
+
+    if (tags.length > 0) {
+      return this.state.defaultImages[tags[0].toLowerCase()];
+    }
+    return this.state.defaultImages.all;
+  }
```

Comment:
Since the images are constant and `getDefaultImage` is a pure function, you can pull both of these out of the `BlogArticle` class and have them be file constants.

```
const IMAGES = {
  all: '/public/images/blogimg_all.png',
  ...
};

function getDefaultImage(tags: Array): String {
  if (tags.length > 0) {
    return IMAGES[tags[0].toLowerCase()];
  }
  return IMAGES.all;
}

class BlogArticle ...
```